<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualTransferMenu</class>
 <widget class="QMainWindow" name="ManualTransferMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>MANUAL ASCII TRANSFER MENU</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileStatusLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPTY</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileTypeLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>备注:</string>
    </property>
   </widget>
   <widget class="TimeLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>1234567890</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START TRANSFER</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
